src/Tasks/LockCheck.cs

```diff
@@ -56,8 +56,10 @@ internal enum ApplicationType
             string[] rgsServiceNames);
 
         [DllImport(RestartManagerDll, CharSet = CharSet.Unicode)]
-        private static extern int RmStartSession(out uint pSessionHandle,
-            int dwSessionFlags, StringBuilder strSessionKey);
+        private static extern int RmStartSession(
+            out uint pSessionHandle,
+            int dwSessionFlags,
+            char[] strSessionKey);
```

Comment:
this doesn't have `[out]`, so I think StringBuilder might be valid.

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/restartmanager/nf-restartmanager-rmstartsession

Reply:
`[Out] char[]` would be the best way to declare the parameter if it wasn't for the strange behavior of the function not null-terminating the string. Please leave as is (i.e. the default `[In,Out]`).